source/includes/steps-create-local-master-key.yaml

```diff
@@ -0,0 +1,68 @@
+title: Create the local master key
+ref: create-local-master-key
+level: 4
+content: |
+  Generate the local master key and save it to a file. You should only need to run this once:
+
+  .. tabs::
+
+     tabs:
+
+       - id: java-master-key-generator
+         name: "Java"
+         content: |
+
+           .. code-block:: java
+
+             import java.io.FileOutputStream;
+             import java.io.IOException;
+             import java.security.SecureRandom;
+
+             public class CreateMasterKeyFile {
+               public static void main(final String[] args) {
+
+                 final byte[] localMasterKey = new byte[96];
+                 new SecureRandom().nextBytes(localMasterKey);
+
+                 String path = "master-key.txt";
+                 try (FileOutputStream stream = new FileOutputStream(path)) {
+                   stream.write(localMasterKey);
+                 } catch (IOException e)  {
+                   e.printStackTrace();
+                 }
+               }
+             }
+
+---
+title: Load the local master key into the client settings 
+ref: load-local-master-key
+level: 4
+content: |
+  Next, in our MongoDB client application, add the following logic to read the master key from the file we created and construct the KMS settings:
```

Comment:
At this point I don't think we've introduced a client application. How do we plan to incorporate this object into the app? Once we know that, we may be able to be opinionated in this step without needing to specify exactly where to run the code.

Maybe this could also output to a file and we could read that file and serialize the KMS map out of it later when we need it?

Reply:
Alternatively, we could break this step out and include it after we've introduced the app

Reply:
To address all three of your comments in this section:

I thought it would be best to keep all the logic of creating and including the local master key in this section rather than returning to reference it later (perhaps in "[Section C. Configure the MongoDB Client](https://github.com/mongodb/docs-ecosystem/co
mmit/4f47fece63aeab1145d7ae5bb8ea6e73ea32d2b6#diff-c5c354e57498148e8ef37f38712ce945R67)") since it is possible to forget content from a couple sections prior, but perhaps that does not match what you were thinking.

Let me know if you have a preference for one of the following:

1) This is not a prerequisite for creating JSON Schema, so it could be swapped with that section. The introduction on keys could be moved into the Introduction section.
2) Relocate this step to Configure the MongoDB Client. The introduction on keys can stay in Create a Local Master Key.

Reply:
Resolved in person, will be creating a new section on setting up encryption client to create data keys.